test/runtests.jl

```diff
@@ -84,10 +84,6 @@ end
     include("standalone/Soil/conservation.jl")
 end
 
-@safetestset "Soil spatial parameters" begin
-    include("standalone/Soil/spatial_parameters.jl")
-end
-
```

Comment:
These two spatial parameters tests (canopy and soil) also got missed when we rebased yesterday, and should be added back